skbio/diversity/beta/_base.py

```diff
@@ -44,15 +47,25 @@ def pw_distances(counts, ids=None, metric="braycurtis"):
 
     See Also
     --------
+    unweighted_unifrac
+    weighted_unifrac
     scipy.spatial.distance.pdist
     pw_distances_from_table
 
     """
+    # this should probably go somewhere else. where?
+    _skbio_metrics = {'unweighted_unifrac': unweighted_unifrac,
+                      'weighted_unifrac': weighted_unifrac}
     num_samples = len(counts)
     if ids is not None and num_samples != len(ids):
         raise ValueError(
             "Number of rows in counts must be equal to number of provided "
             "ids.")
+    if metric in _skbio_metrics:
+        metric = _skbio_metrics[metric]
+
+    if callable(metric):
+        metric = partial(metric, **kwargs)
```

Comment:
Probably not relevant here, but if you don't know about `operator.methodcaller`, it's worth looking at. It's like `partial` except you can have it call a method of an object. Came across it the other day in [pyinstrument](https://github.com/joerick/pyinstrument/blob/master/pyinstrument/profiler.py#L293)